### PR TITLE
I made the responsive state for resolution <= 1024px

### PR DESCRIPTION
**Pull request by seb-wolox**
# Summary
- I created the styles for the webpage

# Trello card
https://trello.com/c/7tYsJDcc/15-dise%C3%B1o-responsive

# Screenshots
Ipad (1024px)
ipad
GalaxyS5 (360px)
screencapture-localhost-63342-sm-express-js-training-maquetado-index-html-2018-06-01-15_03_07

### CODE REVIEW

File: training-maquetado/scss/_navbar.scss [hunk 1]
```diff
@@ -18,3 +18,9 @@
     width: 200px;
   }
 }
+
+@media (max-width: $max-width-responsive) {
```
Comment (meelrossi) [inline training-maquetado/scss/_navbar.scss]: i would always use the same condition. Do not have a min-width and a max-width

File: training-maquetado/scss/_footer.scss [hunk 2]
```diff
@@ -5,23 +5,45 @@
...
+  font-size: $text-xsmall;
+}
+
+@media (min-width: $min-width-responsive) {
+  .footer {
+    padding: 0;
+  }
+
+  .footer-text::after {
+      width: 600px;
```
Comment (meelrossi) [inline training-maquetado/scss/_footer.scss]: identation :D

File: training-maquetado/scss/_navbar.scss [hunk 3]
```diff
@@ -18,3 +18,9 @@
...
+  .navbar {
+    font-size: 0;
```
Comment (meelrossi) [inline training-maquetado/scss/_navbar.scss]: if this is to not show the text, i think it is better to set a display none

File: training-maquetado/scss/_texts.scss [hunk 4]
```diff
@@ -12,6 +12,12 @@
 
 .text-4 {
   color: white;
-  font-size: $text-big;
+  font-size: $text-medium;
   text-transform: uppercase;
 }
+
+@media (min-width: $min-width-responsive) {
```
Comment (meelrossi) [inline training-maquetado/scss/_texts.scss:19]: when adding responsive do not change the current styles to theresponsive ones and add a new media query with the previous ones. Directly set the responsive styles inside the media query with max-width

Comment (seb-wolox) [reply training-maquetado/scss/_texts.scss:19]: I'm not sure if I understand this. But, my idea doing it this way, was to simulate a "mobile first" experience. So if I delete all media queries it'll look like a mobile app  (as suggested in the Grid Video). I could change it if needed :)

**Review by meelrossi — APPROVED** [review]

**Pull request merged**